packages/terra-section-header/tests/wdio/section-header-spec.js

```diff
@@ -1,113 +1,96 @@
 Terra.describeViewports('SectionHeader', ['huge'], () => {
-  describe('Default Section Header', () => {
-    before(() => browser.url('/#/raw/tests/terra-section-header/section-header/default-section-header'));
+  it('should display default Section Header', () => {
+    browser.url('/raw/tests/terra-section-header/section-header/default-section-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('default');
   });
 
-  describe('Long Title Default Section Header', () => {
-    before(() => browser.url('/#/raw/tests/terra-section-header/section-header/long-title-section-header'));
+  it('should display long title Section Header', () => {
+    browser.url('/raw/tests/terra-section-header/section-header/long-title-section-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('long title');
   });
 
-  describe('Long Title Accordion Section Header', () => {
+  describe('Long Title Accordion', () => {
     before(() => {
-      browser.url('/#/raw/tests/terra-section-header/section-header/long-title-accordion-section-header');
+      browser.url('/raw/tests/terra-section-header/section-header/long-title-accordion-section-header');
       browser.moveToObject('#root', 0, 700); // move cursor to remove focus styles
     });
 
-    Terra.it.validatesElement();
+    it('should display long title accordion Section Header', () => {
+      Terra.validates.element('default');
+    });
 
-    describe('Hovered - Long Title Accordion Section Header', () => {
-      before(() => {
-        browser.moveToObject('.accordionContent');
-      });
+    it('should display long title accordion Section Header with hover', () => {
+      browser.moveToObject('.accordionContent');
 
-      Terra.it.validatesElement();
+      Terra.validates.element('hover');
     });
 
-    describe('Keyboard Focus - Long Title Accordion Section Header', () => {
-      before(() => {
-        browser.moveToObject('#root', 0, 700); // move cursor to remove focus styles
-        browser.keys('Tab');
-      });
+    it('should display keyboard focused long title accordion Section Header', () => {
+      browser.moveToObject('#root', 0, 700); // move cursor to remove focus styles
+      browser.keys('Tab');
 
-      Terra.it.validatesElement();
+      Terra.validates.element('keyboard focus');
     });
 
-    describe('Active - Long Title Accordion Section Header', () => {
-      before(() => {
-        browser.moveToObject('.accordionContent');
-        browser.buttonDown();
-      });
+    it('should display active long title accordion Section Header', () => {
+      browser.moveToObject('.accordionContent');
+      browser.buttonDown();
 
-      Terra.it.validatesElement();
-
-      after(() => browser.buttonUp());
+      Terra.validates.element('active');
     });
+    after(() => browser.buttonUp());
```

Comment:
Should this be in the `it` block that executed buttonDown?

Reply:
Updated here https://github.com/cerner/terra-core/pull/3288/commits/77f806dc5902c98d806431f42f9fa799a45649b2